resources

```diff
@@ -0,0 +1,2 @@
+LICENSE:6:- The Camera Icon src/com/t_oster/visicut/gui/ressources/camera.png is from the Token Icon Set of EVAN BROOKS (http://brsev.com/). Its licensed under "Creative Commons Attribution Non-commercial (by-nc)"
```

Comment:
looks like you accidentally added a new file?

Reply:
Yes that was done by mistake.